system/webrtc/tests/test_stream_session.py

```diff
@@ -68,6 +68,8 @@ def test_incoming_proxy(self, mocker):
 
       mocked_pubmaster.reset_mock()
 
+  '''
+  FIXME, hangs for some reason
```

Comment:
usually a skip is added instead. then the test still shows up in collection and you don't have to comment out all the imports

example: https://github.com/commaai/openpilot/blob/541b5cbf1387df07e76da55d3e332170d9e756f0/system/updated/casync/tests/test_casync.py#L16